d2renderers/d2svg/d2svg.go

```diff
@@ -657,19 +661,33 @@ func drawShape(writer io.Writer, targetShape d2target.Shape, sketchRunner *d2ske
 		fmt.Fprintf(writer, closingTag)
 		return labelMask, nil
 	case d2target.ShapeOval:
-		if targetShape.Multiple {
-			fmt.Fprint(writer, renderOval(multipleTL, width, height, style))
-		}
-		if sketchRunner != nil {
-			out, err := d2sketch.Oval(sketchRunner, targetShape)
-			if err != nil {
-				return "", err
+		if !targetShape.DoubleBorder {
+			if targetShape.Multiple {
+				fmt.Fprint(writer, renderOval(multipleTL, width, height, style))
+			}
+			if sketchRunner != nil {
+				out, err := d2sketch.Oval(sketchRunner, targetShape)
+				if err != nil {
+					return "", err
+				}
+				fmt.Fprintf(writer, out)
+			} else {
+				fmt.Fprint(writer, renderOval(tl, width, height, style))
 			}
-			fmt.Fprintf(writer, out)
 		} else {
-			fmt.Fprint(writer, renderOval(tl, width, height, style))
+			if targetShape.Multiple {
+				fmt.Fprint(writer, renderDoubleOval(multipleTL, width, height, style))
+			}
```

Comment:
we should remove this. i don't think it'd ever look good for `multiple` + `double-border`. we just choose one at random and say it takes precedence.